HouseBuilder_Exercise2/src/houseBuilerPackage/House.java

```diff
@@ -1,25 +1,33 @@
 package houseBuilerPackage;
-
+import java.util.ArrayList;
 public class House {
 	
-	private String bathroom;
-	private String bedroom;
+	private ArrayList<String> bathroom;
+	private ArrayList<String> bedroom;
 	private String kitchen;
 	private String livingRoom;
-	private int bathroomNum;
-	private int bedroomNum;
+	private int bathroomCounter;
```

Comment:
Is not necessary to count the items with a variable. You can always return the ArrayList size.